docs/provisioning-methods-and-credentialszip.adoc

```diff
@@ -0,0 +1,63 @@
+// Copy of this topic: https://docs.ota.here.com/concepts/provisioning-methods-and-credentialszip.html adapted for aktualizr repo. Replaces duplicate "credentials.adoc".
+
+= Provisioning methods and credentials.zip
+
+If you're trying to work on integrating {product-name} into your device build, it may be helpful to have a little bit of reference information on how the different provisioning methods work together, and what exactly the magic sauce inside `credentials.zip` is.
```

Comment:
`{product-name}`.
And are we really using the phrase "magic sauce"!? Your call, honestly.

Reply:
Yeah....that's not one of mine. Not very "non-native speaker" friendly, I agree. I will fix that one.